### PR TITLE
Record duration of ./tools/test.py (and optionally ./tools/build.py) on Travis

### LINKED ISSUE #824: Benchmark wanted: time it takes to run tools/test.py
Since tools/test.py will have already run when tools/benchmark.py runs, it should be relatively easy to capture the time it took in .travis.yml and pass it via the command line to tools/benchmark.py.

### PR DESCRIPTION
**Pull request by kevinkassimo**
Closes #824 

Save duration for running `./tools/test.py`. I also added `./tools/build.py`, but could be removed if not needed.

duration

(`build` time on the image is very short since I have built the binary already)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -69,14 +64,13 @@ before_script:
 script:
 - ./tools/lint.py
 - bash -c "sleep 2100; pkill ninja" &
-- TIMEFORMAT="%E" # set time format
-- TRAVIS_BUILD_DURATION=$(time ./tools/build.py -j2 3>&1 1>&2 2>&3) # swap stdout and stderr
-- TEST_DURATION=$(time ./tools/test.py $DENO_BUILD_PATH 3>&1 1>&2 2>&3) # swap stdout and stderr
+- BUILD_DURATION=$(/usr/bin/time -p ./tools/build.py -j2 3>&1 1>&2 2>&3 | awk '/real/ {print $2}') # swap stdout and stderr
```
Comment (piscisaureus) [inline .travis.yml]: That's a pretty mind boggling concoction. Did you consider using tail and using a temp file?

**Comment by ry** [comment]
I prefer #832. Thanks tho @kevinkassimo

**Comment by kevinkassimo** [comment]
@ry I also prefer #832 's handling of overall travis running time, but seems the PR only addresses #822 but not #824

**Comment by ry** [comment]
@kevinkassimo Oh yea - sorry I missed that. You you please open another PR for it? And I echo @piscisaureus's concerns about the mind boggling concoction - it would be nice if you could find simpler way to do that.

**Pull request closed without merging**